pcdsdevices/areadetector/detectors.py

```diff
@@ -264,14 +277,31 @@ class PCDSAreaDetectorTyphosBeamStats(PCDSAreaDetectorTyphos):
     target_y = Cpt(EpicsSignalWithRBV, 'Cross4:MinY', kind='normal')
 
 
-class PCDSAreaDetectorTyphosTrigger(PCDSAreaDetectorTyphos):
+class BaslerBase(Device):
     """
-    Expanded typhos-optimized areadetector class for cameras with triggers.
+    Base class with Basler specific PVs. Intended to be sub-classed, not used
+    stand-alone. 
     """
+    reset = Cpt(EpicsSignal, 'RESET.PROC', kind='config', doc='Reset the camera')
+    set_metadata(reset, dict(variety='command-proc', value=1))
+    packet_size = Cpt(EpicsSignal, 'GevSCPSPacketSiz_RBV',
+                      write_pv='GevSCPSPacketSiz', kind='config',
+                      doc='Set Enet Packet Size (typ. 9000)')
+    enet_bw = Cpt(EpicsSignalRO, 'GevSCDCT_RBV', kind='config',
+                  doc='Current Enet BW')
 
-    event_code = Cpt(EpicsSignalWithRBV, 'CamEventCode', kind='config',
-                     doc='Code to determine beam synchronization rate.')
-    event_rate = Cpt(EpicsSignalRO, 'CamRepRate_RBV', kind='config',
-                     doc='Current rate of the incoming triggers. '
-                         'Determined by event_code and the '
-                         'accelerator state.')
+
+# Typical "hutch" Basler class
+class Basler(PCDSAreaDetectorTyphosTrigger, BaslerBase):
+    """
+    Class for Basler cameras.
+    """
```

Comment:
```suggestion
    See Also
    --------
    :class:`LasBasler`
        Basler camera with additional laser-specific entries.
    """
```

Could be worthwhile if this ever gets moved around